src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,15 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
-        return false;
+        if (a>=b && b>=c || a<b && b<c || a>c && b<c) return true;
```

Comment:
try to follow the way, when first term in clause would be term, that you actually need to check. In our case - it's `b`, so, it should be: `b<=a && b>=c || b>.....` - in such way it'll be easier to determine what actually should be checked.
Also there is no need for `if-else` clause here, as your expression will return `boolean` - try to solve this task without `if-else` clauses.